src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3866,35 +3983,43 @@ class CgroupUtils(object):
             else:
                 avail_resc = self.available_node_resources(node)
             assigned = self.assign_job(hostresc, avail_resc, node)
+            if attempt != 0:
```

Comment:
Is there a reason why we're stopping here if it failed?

Reply:
I added a comment to the code.